Day1Main.py

```diff
@@ -0,0 +1,25 @@
+from InputReader import InputReader
+from Day1.DayOnePartOne import FloorCounter
+from Day1.DayOnePartTwo import BasementFinder
+
+input_reader = InputReader("input1.txt")
+raw_input = input_reader.get_raw_input()
+
+floor_counter = FloorCounter(raw_input)
+parsed_input = floor_counter.parse_input()
+floor_counter.count_floors(parsed_input)
+print("The final floor at the end of the input:", floor_counter.get_final_floor())
+
+basement_index_finder = BasementFinder(raw_input)
+basement_index_finder.calculate_basement_entry(parsed_input)
+print("The step at which Santa enters the basement:", basement_index_finder.get_basement_entry())
+
+robs_challenged1 = FloorCounter("(((-2)+6((")
```

Comment:
Ez nagggyon nem bueno floors counter kap egy random gibrisht bemenetnek es elnevezd robs challengednek 

Rob is challanged tru dat, de adsz megint egy statikus nevet {aminek koze sincs ahhoz amit takar) es utana csinaltatsz vele vmit. statikus nevek nem csinalnak semmit! azok statikusak,  ahogy beszelt nyelvekben is:  ebedelek nem ebed teljesen mast jelent a ketto

A floorcounter az floorcounter nem mas,az egyenloseg ket oldalan legalabb kb ugyanaz legyen.  ranezesbol kb be kell tudom loni mi az
Ennel amugy meg a rob_challenged_floor_counter is jobb

Az egyes a vegen nem tudom mit akart jelezni :D mi a kulonbseg 1-2-19047 kozott? Hany van? 

A floor counter bemenete legyen meaningfull tedd be egy valtozoba mini_input v mini_raw_input v barmi 

A floor counter class ne kapjon raw inputot!
A legjobb ha semmit nem kap es a fvnek ado,d mert a classhoz nem kell az inpu,t anelkul is ertelmes egesz. a fv vesziti el az ertelmet ha nincs min dolgoznia neki kell. Amit a classnak akarsz adni azok konstans nem valtozo dolgok: van egy map classod aminek van egy get capital of country fve
Itt a classnak lehet egy country to capital dictionaryje amit megkap ctorba es a fv meg megkapja a kerdeses orszagot. A parositas nagyon ritkan valtozik nincs ertelme mindig beadni a fvnek uh elteszem a classba. Az h melyik orszagot kerdezem vszinu hivasonkent valtozik uh megkapja a fv. Nem hozok letre egy uj classt minden kerdesnel